sentry.client.config.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "",
```

Comment:
Note that I didn't include the DSN, I'm pretty sure it's safe to just land (e.g. if clients are sending errors to the same env), but we could put it in an env var and we probably want to tunnel through our own server-side route anyway, so we don't need to hardcode it here in any case. For the moment you do need to fill it in to test though.

Reply:
Looking at [our current config](https://github.com/mozilla/blurts-server/blob/326a24c225ac94fd419d085fa86cb0acd7cd3a36/src/app.js#L30-L44), we already have an env var `SENTRY_DSN` that we could use?

(Although if this script gets run on the client-side, it might [need to be prefixed `NEXT_PUBLIC`](https://nextjs.org/docs/app/building-your-application/configuring/environment-variables#exposing-environment-variables-to-the-browser).)